core/src/Cuda/Kokkos_Cuda.hpp

```diff
@@ -166,8 +166,17 @@ class Cuda {
 
   Cuda();
 
-  Cuda(cudaStream_t stream,
-       Impl::ManageStream manage_stream = Impl::ManageStream::no);
+  explicit Cuda(cudaStream_t stream) : Cuda(stream, Impl::ManageStream::no) {}
+
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
+  template <typename T = void>
+  KOKKOS_DEPRECATED_WITH_COMMENT(
+      "Cuda execution space should be constructed explicitly.")
```

Comment:
You are correct.

Reply:
Oh wait, sorry I read this backwards. I used "should" because if they have Deprecated code turned on it's technically not a "must" that they use the correct construction. But it is a must after v4. So I don't mind changing them.

Reply:
Keep it as is then